test/03-date-tests.js

```diff
@@ -86,7 +86,7 @@ describe('03-date-tasks', function() {
             }, {
                 startDate: new Date(2000,1,1,10,0,0),
                 endDate:   new Date(2000,1,2,15,20,10,453),
-                expected:  '29:20:10.453'
```

Comment:
Please, revert this changes

Reply:
fixed, I recently saw correct date difference